GooglePay/src/main/java/com/braintreepayments/api/GooglePayClient.java

```diff
@@ -182,39 +181,38 @@ public void createPaymentAuthRequest(@NonNull final GooglePayRequest request,
         braintreeClient.sendAnalyticsEvent("google-payment.selected");
 
         if (!validateManifest()) {
-            callback.onResult(null, new BraintreeException(
-                    "GooglePayActivity was " + "not found in the " + "Android " +
-                            "manifest, or did not have a theme of R.style.bt_transparent_activity"));
+            callback.onGooglePayPaymentAuthRequest(new GooglePayPaymentAuthRequest.Failure(new BraintreeException(
+                    "GooglePayActivity was not found in the Android " +
+                            "manifest, or did not have a theme of R.style.bt_transparent_activity")));
```

Comment:
this is _oddly_ specific heh. No action, just commenting

Reply:
Yeah... it is getting pretty verbose open to alternative callback naming! The previous pattern was always naming it `onResult` which is a little confusing when the result we are returning is actually a request object